master/reference/public-cloud/aws.md

```diff
@@ -46,27 +46,15 @@ See the [IP pool configuration reference]({{site.baseurl}}/{{page.version}}/refe
 for information on how to configure Calico IP pools.
 
 By default, Calico's ipip encapsulation applies to all container-to-container traffic.  However,
-encapsulation is only required for container traffic that crosses a VPC subnet boundary.  For better performance, you can
-configure Calico to perform ipip encapsulation only across VPC subnet boundaries. To do this, you must
-modify Calico's BIRD config.
-
-Modify bird_ipam.cfg.template [found here](https://github.com/projectcalico/calicoctl/blob/master/calico_node/filesystem/etc/calico/confd/templates/bird_ipam.cfg.template).
-
-Configure the calico_ipip filter as shown below:
-
-```
-filter calico_ipip {
-  if ( gw ~ <instance_private_ip>/<vpc_subnet_prefix_len> ) then {
-    accept;
-  } else {
-    krt_tunnel = "tunl0";
-    accept;
-  }
-}
-```
-
-Enabling ipip encapsulation allows container-to-container traffic to pass VPC subnet
-boundaries, while outgoing NAT allows containers to access EC2 instances across VPC subnet boundaries.
+encapsulation is only required for container traffic that crosses a VPC subnet boundary.  For better 
+performance, you can configure Calico to perform ipip encapsulation only across VPC subnet boundaries.  
+To do this ensure your node resource is correctly configured with the IPv4 address and subnet of each 
+host (you can verify using `calicoctl get nodes`), and set the `mode` field in the `ipip` section of the
+IP pool configuration to be `cross-subnet`.
```

Comment:
Please can you spell this out with an example.  "For example, suppose you have two VPCs with <correct AWS term for the IP pool>s configured as 10.1.0.0/16 and 10.2.0.0/16.  Node in VPC 1 should be configured like this: <sample yaml>.  If you have old config, it might look like this: ...."